src/tutorials/lua_basics_lightbulb.md

```diff
@@ -365,398 +390,227 @@ switchTrigger.interactedEvent:Connect(OnSwitchInteraction)
+        Switch:RotateTo(ROTATION_OFF, TIME_ROTATE, true)
 ```
 
 - `else` is used with `if` to tell the script if the if conditions are not true, do the following instead.
-- `switch:RotateTo()` tells the script to rotate our switch variable. The switch needs to rotate to its original downwards position, since we defined its initial rotation as `switchStartingRotation`, we can simply plug `switchStartingRotation` into our rotation statement.
-- `0.5` is the time in seconds it takes to complete the action.
 
-Your `OnSwitchInteraction()` function should now look like this:
+Your `OnTriggerInteracted()` function should now look like this:
 
 ```lua
-local function OnSwitchInteraction()
-    if not lightIsOn then
-        switch:RotateTo(onRotation, 0.5, true)
-        World.SpawnAsset(propLightTemplate, {position = bulbPosition})
+function OnTriggerInteracted()
+    -- Flips the cached state of if the switch is on or off
+    isSwitchOn = not isSwitchOn
+
+    if isSwitchOn then
+        Switch:RotateTo(ROTATION_ON, TIME_ROTATE, true)
+        Light.visibility = Visibility.INHERIT
     else
-        switch:RotateTo(offRotation, 0.5, true)
+        Switch:RotateTo(ROTATION_OFF, TIME_ROTATE, true)
     end
-
-    lightIsOn = not lightIsOn
 end
 ```
 
-Press **Play** to see if your `else` statement works. The switch should now rotate up when first interacted with, then down on your second interaction with it. However, we still need to de-spawn (delete) the light when the light switch is turned off.
-
----
+Press **Play** to see if your `else` statement works. The switch should now rotate up when first interacted with, then down on your second interaction with it. However, we still need to turn off the light when the light switch is turned off.
 
 ## Turning the Light Off
 
-### Make a Variable for the Spawned Light
-
-In order to turn off the light, you first need to define the light after it is spawned. When the **LightTemplate** is spawned, it shows up at the bottom of the Hierarchy. Make a variable to reference this at the top of the script:
-
-```lua
-local spawnedLight = nil
-```
+Your next project goal is to make the switch more functional and have it turn on a light when players interact with it.
 
-- `spawnedLight` is the name you are giving to the light that spawns.
-- `nil` means that it has no value yet.
-
-Next, in the line where you spawn the light, assign what is spawned to the variable, by adding `spawnedLight =` to the beginning:
+In the `OnTriggerInteracted()` in the else statement under the `RotateTo()` function, add:
 
 ```lua
-    spawnedLight = World.SpawnAsset(propLightTemplate, {position = bulbPosition})
+Light.visibility = Visibility.FORCE_OFF
 ```
 
-- `World.SpawnAsset` is doing two different things here. It creates the asset in the game space, and also **returns** the unique ID of the asset that just got spawned, which you are saving to the variable `spawnedLight`.
-
-### Destroy the Light
+- `FORCE_FF` disables visibility.
```

Comment:
missing a letter here.

Also make clear that this means that it makes it not visible.